spec/02-integration/03-admin_api/09-targets_routes_spec.lua

```diff
@@ -320,4 +320,52 @@ describe("Admin API", function()
       end)
     end)
   end)
+
+  describe("/upstreams/{upstream}/targets/{target}", function()
+    describe("DELETE", function()
+      local target
+
+      before_each(function()
+        assert(helpers.dao.targets:insert {
```

Comment:
shouldn't an upstream be created here? reusing an existing one might interfere with the test results here.